core/domain/topic_domain.py

```diff
@@ -221,14 +236,107 @@ def validate(self):
                     % (skill_id, self.id))
 
 
+class SubtopicPage(object):
+    """Domain object for a Subtopic page."""
+
+    def __init__(self, subtopic_page_id, topic_id, html_data, language_code):
+        """Constructs a SubtopicPage domain object.
+
+        Args:
+            subtopic_page_id: str. The unique ID of the subtopic page.
+            topic_id: str. The topic that this subtopic is a part of.
+            html_data: str. The html data that describes this subtopic.
+            language_code: str. The ISO 639-1 code for the language this
+                sub topic page is written in.
```

Comment:
subtopic

This is interesting though. Can we modify the html_data schema to use something that's more like SubtitledHtml instead? This is "learning content" and will probably need to have audio subtitles for learners whose first language is not English.

Might need to extract SubtitledHtml into a generalized thing somewhere, too, but I'm not sure -- it depends on how much similarity there actually is. (@DubeySandeep is making changes to the one in Exploration now, so you may want to talk with him and see if the new design applies well to this use case, or if we should just go our own way here and have no reuse.)